homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+  &-inner{
+    @extend %flex-primary;
+    justify-content: space-between;
+    background: transparent;  
+  }
+  .logo-link{
+    position: relative;
+    color: $color-text-header;
+    font-family: $font-hero-title, sans-serif;
+    font-size: 24px;
+    font-weight: bold;
+    line-height: 34px;  
+    letter-spacing: 2px;
+    text-transform: uppercase;      
+  }  
+  .header-button{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 110px;
+    max-width: 180px;
+    width: 100%;
+    & .btn{
+      color: $color-button;
+      border: 1px solid $button-border-color;
+      width: 100%;
+    }
+    & .get-btn{
+      margin-right: 0;
+    }
+    & .log-btn{
+      max-width: 90px;
+      margin-top: 16px;
+    }
+  }
+}
+.sticky-header {
+  background: $bg-sticky-header;
+  & .logo-link{
+    color: $sticky-logo;
+  }
+  & .hamburger-menu::before, 
+  & .hamburger-menu::after, 
+  & .hamburger-menu span {
+      background-color: $bg-sticky-hamburger-menu;
+  }
+}
+.hamburger-menu{
+  position: relative;
+  display: block;
+  height: 18px;
+  width: 24px;
+  cursor: pointer;
+  &::before,
+  &::after,
+  & span{
+    position: absolute;
+    left: 0;
+    height: 10%;
+    width: 100%;
+    background-color: $bg-color-primary;
+    transition: all 0.3s ease 0s;
+  }
+  &::before,
+  &::after{
+    content: '';
+  }
+  &::before{
+    top: 0;
+  }
+  &::after{
+    bottom: 0;
+  }
+  & span{
+    top: 50%;
+    transform: scale(1) translate(0px, -50%);
+  }
+  &.active span{
+    transform: scale(0) translate(0px, -50%);
+  }
+  &.active::before{
+    top: 50%;
+    transform: rotate(-45deg) translate(0px, -50%);
+  }
+  &.active::after{
+    bottom: 50%;
+    transform: rotate(45deg) translate(0px, 50%);
+  }
+}
+.menu-body{
+  position: absolute;
+  top: 70px;
+  left: 0;
+  width: 100%;
+  z-index: 1;
+  background-color: $bg-color-primary;
+  transition: transform .5s ease-out;
+  transform: translateY(-130%);
+  .menu-list{
+    position: relative;
+    @extend %flex-primary;
+    flex-direction: column;
+    width: 100%;
+    padding-top: 40px;  
+  }
+  &.active{
+    display: flex;
+    flex-direction: column;    
+    height: 100vh;    
+    align-items: center;
+    transform: translateY(0px);
+  }
+  & .menu-link{
+    position: relative;
+    display: inline-block;
+    padding: 6px 0;
+    font-family: $font-links, sans-serif;
+    color: $color-link-header-mobile;
+    font-size: 12px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    cursor: pointer; 
+  }
+  & .item-active.menu-link::after{
+    width: 50%;
+  }
+  & .menu-item .menu-link:after{         
+    content: '';
+    position: absolute;
+    left: 50%;
+    display: block;
+    height: 2px;
+    width: 0%;
+    background-color: $bg-underline-link;   
+    -webkit-transition: width 0.3s ease-in-out;
+    -moz--transition: width 0.3s ease-in-out;
```

Comment:
Prefixes not needed in scss. They should be added automatically

Reply:
done